### PR TITLE
Fix/165454813 update rn libs dependencies

### PR DESCRIPTION
**Pull request by diablourbano**
This PR updates/upgrades libraries reported after PillarWallet was upgraded to RN0.59.

```
- react-native-intercom
- react-native-randombytes
- react-native-sentry
- react-native-fetch-blob
- react-native-camera
- react-native-color-matrix-image-filters
- react-native-vector-icons
- react-native-image-crop-picker
- react-native-device-info
- react-native-background-timer
- react-native-os
- react-native-scrypt
- react-native-tcp
- react-native-linear-gradient
- react-native-udp
```

You can check on [pivotal ticket](https://www.pivotaltracker.com/story/show/165454813) details on some libraries that were already on latest version.

Tested on iOS and android.

### CODE REVIEW

File: ios/sentry.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
 defaults.url=https://sentry.io/
 defaults.org=pillar-project-worldwide-ltd
 defaults.project=rn-pillar-wallet
-auth.token=d63aa51d77ae45fea95a230c94aae4a910b5e8613b7745ca9a545f143728476f
+auth.token=d33bb0eb5dc4485c9c97450a328c39440542b6c11ead4c8099d20257b5513940
```
Comment (grilix) [inline ios/sentry.properties]: Just checking, do we really want to change this?

File: ios/pillarwallet/AppDelegate.m [hunk 2]
```diff
@@ -27,7 +27,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
       [Intercom setApiKey:@"ios_sdk-f210e1d785d4c0e64ab3ba0f529d64c47da59186" forAppId:@"s70dqvb2"];
       jsCodeLocation = [[NSBundle mainBundle] URLForResource:@"main" withExtension:@"jsbundle"];
     #endif
-    RCTRootView *rootView = [[RCTRootView alloc] initWithBundleURL:jsCodeLocation
+RCTRootView *rootView = [[RCTRootView alloc] initWithBundleURL:jsCodeLocation
```
Comment (grilix) [inline ios/pillarwallet/AppDelegate.m]: This indentation looks weird

File: ios/sentry.properties (hunk 1 above)
Comment (diablourbano) [reply ios/sentry.properties]: That's done by sentry wizard during the linking process, however I already asked on Pillar channel if this changes anything and no, besides, the tests were done on ios and android and everything works as expected

Comment (diablourbano) [reply ios/sentry.properties]: That's done by sentry wizard during the linking process, however I already asked on Pillar channel if this changes anything and no, besides, the tests were done on ios and android and everything works as expected

File: ios/pillarwallet/AppDelegate.m (hunk 2 above)
Comment (diablourbano) [reply ios/pillarwallet/AppDelegate.m]: done

**Pull request merged**